### DISCUSSION TITLE
How to unactivate autostart?

### DISCUSSION THREAD

**Discussion by Golddouble** [evilsocket/opensnitch#861] [Q&A] [answered]
Hello,

How can unactivate autostart for opensnitch?

Is there an easy simple way?

Version Opensnitch 1.5.2
MX-19 (Xfce, debian10)

I have tried to unactivate autostart for opensnitch this way:
settings-> session and startup-> application autostart
But this was not working.

Thank you.

**Comment by gustavo-iniguez-goya** [comment, marked as answer]
Hi @Golddouble ,

Remove this file from your home: `~/.config/autostart/opensnitch_ui.desktop`

**Reply by BobSquarePants** [reply]
FYI, This will only stop the GUI but not the daemon, Am I right ?

**Reply by gustavo-iniguez-goya** [reply]
correct @BobSquarePants .

If you want to disable the daemon, you'll need to disable the systemd-service: `systemctl disable opensnitch`

**Reply by Golddouble** [reply]
But when I have a SysVinit - system?
Does the command `systemctl disable opensnitch` nevertheless work and disable the daemon?

**Reply by gustavo-iniguez-goya** [reply]
mmmh, try: `update-rc.d opensnitch disable`

https://manpages.debian.org/bullseye/init-system-helpers/update-rc.d.8.en.html
